### ISSUE TITLE
关于WIoU损失函数

### ISSUE THREAD

**Issue by HornGate** [z1069614715/objectdetection_script#18]
elif scale:
                    return getattr(WIoU_Scale, '_scaled_loss')(self), (1 - iou) * torch.exp((rho2 / c2)), iou # WIoU https://arxiv.org/abs/2301.10051
                else:
                    return iou, torch.exp((rho2 / c2)) # WIoU v1

您好，请问两个问题：1.为什么返回多个iou损失？2.iou的均值iou_mean在WIoU_Scale里是怎么计算的？

**Comment by z1069614715** [comment]
1. 因为后面还要用到iou
2. 这个详细请看源码哈

**Comment by HornGate** [comment]
> 1. 因为后面还要用到iou
> 2. 这个详细请看源码哈

@classmethod
    def _update(cls, self):
        if cls._is_train: cls.iou_mean = (1 - cls._momentum) * cls.iou_mean + cls._momentum * self.iou.detach().mean().item()

请问这个类方法的cls是怎么来的？代表什么意思呢？iou_mean为什么要这么计算呢？
